src/index.ts

```diff
@@ -188,3 +188,6 @@ namespace Experimental {
 }
 
 Error.stackTraceLimit = 100000;
+
+// export parts of the low-level bindings interface for advanced users
+export * as Bindings from './bindings/index.js';
```

Comment:
IMO "bindings" is not the best name/concept for what you're exporting here. So far, I see finite field and elliptic curve primitives. how about exporting them as part of the `Crypto` namespace instead? (currently used for elliptic curve parameters)

The fact that this code resides in the `/bindings` folder is more of a historical artifact. Arguably the crypto stuff (and some of the other TS code in bindings) should live somewhere else, and `/bindings` just contain the various layers of Ocaml and Rust bindings

Reply:
> IMO "bindings" is not the best name/concept for what you're exporting here

Yep, we are still looking for a better name. The math and crypto exports are just the beginning, I imagine that we could export other things in the future too